db/migrate/20240609104039_create_solid_queue_tables.solid_queue.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+# This migration comes from solid_queue (originally 20231211200639)
+class CreateSolidQueueTables < ActiveRecord::Migration[7.0]
+  def change
+    create_table :solid_queue_jobs do |t|
```

Comment:
Solid Queue comes with a dedicated command to clear finished jobs: `SolidQueue::Job.clear_finished_in_batches`. It is currently not run automatically. Therefore, I would suggest to schedule this through `config/schedule.rb` (i.e., daily).

Reply:
What if we set `preserve_finished_jobs=false` instead? I don't see a reason why we should keep these jobs around.

Reply:
We don't necessarily need the job details, that's right. However, removing finished jobs will, potentially, also reduce the monitoring surface we have and the usefulness of the [mission_control-jobs](https://github.com/rails/mission_control-jobs) plugin, I could assume.

Reply:
That's true. I added `SolidQueue::Job.clear_finished_in_batches` (and tested it locally)

Reply:
Thanks :+1: Have you tested the crontab integration? It is just a DSL to edit the crontab file, and at least on my system this currently fails. You can preview the crontab file being written with `whenever` (more options with `whenever --help`, docs [here](https://github.com/javan/whenever)).

The resulting command _should_ work given the environment variables printed as well.

Reply:
Right, I just assumed the code would be executed as Ruby code. It should be fixed now. The cronfile is being generated and when I execute the command in the cronfile, the old jobs are being deleted.

Reply:
Awesome, thanks! This looks good (without testing it myself), and I would assume it's fine :+1: